pep/app/constants/themes.ts

```diff
@@ -57,4 +59,40 @@ export const THEME_TOMATO: Theme = {
     }
 };
 
-export default [THEME_DEFAULT, THEME_TOMATO];
+export const THEME_DARK: Theme = {
+    id: ThemeId.DARK,
+    cssPath: '/assets/themes/dark.css',
+    label: 'theme.themes.dark',
+    colors: {
+        links: {
+            glossary: '#a4a29e',
```

Comment:
this is fine for now, if we get the approval to go the SASS-only route for themes, I think it would be good to eventually move these color values into the SASS theme files too as variables. And then create associated classes for them (e.g. `.link-glossary`, etc) which map to those colors in the base SASS files, and finally switch the stuff that uses this to apply those classes instead.

That way all theme related stuff is completely self contained in the SASS, and don't to worry about any less than ideal dynamic style block/inline styles implementations, and any other deps doing that in-JS might incur.

Reply:
@billdami agreed